infra/modules/monitoring/main.tf

```
@@ -0,0 +1,69 @@
+# Create SNS topic for all email and external incident management tools notifications
+
+resource "aws_sns_topic" "this" {
+  name = "${var.service_name}-monitoring-notifications"
+
+  # checkov:skip=CKV_AWS_26:SNS encryption for alerts is unnecessary 
+}
+
+# Create CloudWatch alarms for the service
+
+resource "aws_cloudwatch_metric_alarm" "high_http_target_5xx_error_count_threshold" {
```

maybe find/replace error, the resource shouldn't be called threshold.

also nit: http 5XXs are always errors, so "error" is a bit redundant.

and final nit: "target" doesn't have a meaning without the context of load balancer, so either call this "load_balancer_target" or if we want to keep it short we can call it "application" or "app"

```suggestion
resource "aws_cloudwatch_metric_alarm" "high_app_http_5xx_count" {
```